cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1648,6 +1661,60 @@ String zkConnectionString(int connectToReplicas)  {
                     });
         }
 
+        Future<ReconciliationState> kafkaBrokerDynamicConfiguration() {
+            KafkaBrokerConfigurationHelper kbch = new KafkaBrokerConfigurationHelper(kafkaSetOperations.getAdminClientProvider(), secretOperations);
+            return kafkaSetOperations.getAsync(namespace, KafkaCluster.kafkaClusterName(name))
+                .compose(sts -> {
+                    if (sts == null) {
+                        return Future.succeededFuture();
+                    }
+                    int replicas = kafkaCluster.getReplicas();
+                    List<Future> configFutures = new ArrayList<>(replicas);
+
+                    for (int podId = 0; podId < replicas; podId++) {
+                        int finalPodId = podId;
+                        Future<Admin> acFut = kbch.adminClient(sts, podId);
+                        configFutures.add(
+                                acFut.compose(ac -> {
+                                    if (ac == null) {
+                                        kafkaPodsUpdatedDynamically.put(finalPodId, null);
+                                        log.warn("Could not create admin client.");
```

Comment:
If this log message should stay here, it should use the `reconciliation` as the prefix.